kie-server-parent/kie-server-services/kie-server-services-openshift/src/main/java/org/kie/server/services/openshift/api/KieServerOpenShift.java

```diff
@@ -15,29 +15,43 @@
 
 package org.kie.server.services.openshift.api;
 
+import java.util.List;
 import java.util.Optional;
 
 import io.fabric8.kubernetes.api.model.ConfigMap;
 import io.fabric8.openshift.api.model.DeploymentConfig;
 import io.fabric8.openshift.client.OpenShiftClient;
 import org.kie.server.services.impl.KieServerLocator;
 
+import static org.kie.server.services.openshift.api.KieServerOpenShiftConstants.CFG_MAP_LABEL_SERVER_ID_KEY;
+
 public interface KieServerOpenShift {
 
     default boolean isKieServerReady() {
         return KieServerLocator.getInstance().isKieServerReady();
     }
 
     default boolean isDCStable(DeploymentConfig dc) {
-        return "True".equals(dc.getStatus().getConditions().get(0).getStatus()) && dc.getStatus().getUnavailableReplicas() == 0;
+        return "True".equals(dc.getStatus().getConditions().get(0).getStatus()) 
```

Comment:
any reason you use mixed case and case sensitive equals here?

Reply:
If we use 'oc' command login to any running OpenShift cluster, then run the following command:
`
$ oc explain DeploymentConfig.status.conditions.status
FIELD: status <string>

DESCRIPTION:
     Status of the condition, one of True, False, Unknown.
`
The possible value of the string literal listed are '**True**', '**False**', '**Unknown**', which is a bit of unusual for Java developers.

Reply:
I was more thinking why not to use equalsIgnoreCase just to be on safe side so to say

Reply:
To be cautious is always good. Changed it to use 'equalsIgnoreCase'. Who knows OpenShift may change its spec all of sudden in future to align with what developer get used to, like 'true' and 'false'.

Reply:
@mswiderski @sutaakar 
Shall we merge it or still have more things to look into?